WeatherStation/v2/_summit.yaml

```diff
@@ -0,0 +1,2 @@
+host: '139.229.170.220'
```

Comment:
Change to something like:
```
instances:
  - sal_index: 1
    lsst_config:
      host: 139.229.170.220
```
(replace `lsst_config` with `config` if using weather-station-class-specific config schema)

Reply:
Done